src/components/Routes.js

```diff
@@ -20,20 +20,20 @@ import { Thread } from './Thread';
 
 export const Routes = withRouter((props) => {
   // Retrieve authorization config from the server so we can build auth URIs
-  var { loading, error, data } = useQuery(FUSIONAUTH_CONFIG,
+  const { loading, error, data } = useQuery(FUSIONAUTH_CONFIG,
   {
     onError(error) {
        errorHandler(error, props.history)
     }
   })
   if (loading) {
     return <p>Loading</p>
-  } else if (error) {
+  } if (error) {
     return <p>An unexpected error occurred, please come back later</p>
   }
   if (loading) {
```

Comment:
Is this `if (loading)`... code duplicated? If so, may be worth cleaning up while you're making changes here.

Reply:
FYI, this pattern will most likely change.